.vscode/settings.json

```diff
@@ -1,5 +1,9 @@
 {
   "typescript.tsdk": "node_modules\\typescript\\lib",
   "cSpell.words": ["fluentui", "reactflow"],
-  "eslint.validate": ["json"]
+  "eslint.validate": ["json"],
+  "terminal.integrated.env.windows": {
```

Comment:
Do we need to remove this?

Reply:
This was accidental. Thanks for pointing it out.